docs/source/settings.rst

```diff
@@ -124,4 +124,44 @@ SPIDERMON_ADD_FIELD_COVERAGE
 ----------------------------
 Default: ``False``
 
-When enabled, Spidermon will add field coverage information in Spider stats.
+When enabled, Spidermon will add statistics about the number of items scraped and coverage for each existing
+field following the format:
+
+``'spidermon_item_scraped_count/<item_type>/<field_name>': <coverage>``
+``'spidermon_field_coverage/<item_type>/<field_name>': <item_count>``
+
+.. note::
+
+   Nested fields are also supported. For example, if your spider returns these items:
+
+   .. code-block:: python
+
+      [
+        {
+          "field_1": {
+            "nested_field_1_1": "value",
+            "nested_field_1_2": "value",
+          },
+        },
+        {
+          "field_1": {
+            "nested_field_1_1": "value",
+          },
+          "field_2": "value"
+        },
+      ]
+
+   Statistics will be like the following:
+
+   .. code-block:: python
+
+      'spidermon_item_scraped_count/dict': 2,
+      'spidermon_item_scraped_count/dict/field_1': 2,
+      'spidermon_item_scraped_count/dict/field_1/nested_field_1_1': 2,
+      'spidermon_item_scraped_count/dict/field_1/nested_field_1_2': 1,
+      'spidermon_item_scraped_count/dict/field_2': 1,
+      'spidermon_field_coverage/dict/field_1': 1,
+      'spidermon_field_coverage/dict/field_1/nested_field_1_1': 1,
+      'spidermon_field_coverage/dict/field_1/nested_field_1_2': 0.5,
+      'spidermon_item_scraped_count/dict/field_2': 0.5,
```

Comment:
This covers “regular” nested fields. What about [nested fields within lists](https://github.com/scrapinghub/spidermon/pull/263#pullrequestreview-459565813)? (if they are not supported, maybe we can just state so).

Reply:
They are not supported (and I don't think it is worthy to add try to add it now). I will include a note on the documentation about this.